test/test_xvfb_browser.py

```diff
@@ -0,0 +1,38 @@
+import os
+from functools import partial
+from typing import List
+
+from openwpm.command_sequence import CommandSequence
+from openwpm.commands.types import BaseCommand
+from openwpm.task_manager import TaskManager
+
+from .openwpmtest import OpenWPMTest
+from .utilities import BASE_TEST_URL
+
+
+class ExceptionCommand(BaseCommand):
+    def __init__():
+        None
```

Comment:
```suggestion
```
Why declare the `__init__` if you don't need it?

Reply:
It was a placeholder--sorry.